application.md

```diff
@@ -1,153 +1,178 @@
+
+OpenAstronomy would like to be a part of Google Summer of Code to reach a new
+pool of contributors, develop our communities, and help the member projects
+collaborate better.
+Some of our member projects' past experience with GSoC shows that it provides a
+boost to areas which are in need of intensive attention, as well as adding
+members to the community who can turn out to be valuable long-lasting
+contributors.
+We would like to help bring these experience to a wider variety of projects, all
+united by the goal of fostering open source development of astronomy software.
 
-This organisation is new and this is our first year applying as such, however,
-two of our sub-organisations (Astropy and SunPy) have participated in GSoC under
-the Python Software Foundation umbrella, both in 2013 and 2014.
-
-## Why is your organization applying to participate in Google Summer of Code 2015? What do you hope to gain by participating?
-
-OpenAstronomy is a collection of projects united by a common goal to foster open
-source development of astronomy software. 
 We are submitting this application together in order to strengthen collaboration
 between the different projects, for example by offering project ideas that
 extend across multiple packages, in order to reduce duplication of efforts and
-strengthen interoperability. 
-As an umbrella organization separate from the Python Software Foundation, we
-will be able to include other projects that are not exclusively Python-based and
-accept students with experience with different languages, while uniting projects
-with the common goal of providing tools for the scientific exploration of the
-Universe.
-
-By participating in GSoC as a group, OpenAstronomy aims to improve the
+strengthen interoperability.
+As an umbrella organization with an astronomy theme, we will be able to include
+diverse projects (not only Python-based) and accept students with experience
+with different languages, while uniting projects with the common goal of
+providing tools for the scientific exploration of the Universe.
+
+By participating in GSoC as an umbrella, OpenAstronomy aims to improve the
 experience for both students and mentors alike, while reducing the
-administrative overhead. 
-Both the SunPy and Astropy projects have participated in GSoC in 2013 and 2014
+administrative overhead.
+Both the SunPy and Astropy projects have participated in GSoC from 2013 to 2015
 under the Python Software Foundation umbrella organization, and the experience
-was highly successful for both organisations (a total of 4 and 10 students
-completed their projects during the 2013 and 2014 programs, many of them with
-exceptional performance, and no students were failed). 
+was highly successful for both organisations (a total of 12 students completed
+their projects during these years, many of them with exceptional performance).
+Additionally the yt project participated last year under NumFocus with positive
+results. 
```

Comment:
I just double-checked with Matt, who confirms that while we had ideas with the NumFocus org, no student actually participated with us. I think that means you should delete this sentence.